joint_limits_enforcement_plugins/src/ruckig_joint_limiter.cpp

```diff
@@ -170,7 +212,7 @@ bool RuckigJointLimiter<joint_limits::JointLimits>::on_enforce(
       ruckig_output_->new_acceleration.at(joint));
   }
 
-  return result == ruckig::Result::Finished;
+  return (result == ruckig::Result::Working || result == ruckig::Result::Finished);
```

Comment:
is it Ok if it is still working and we return? Is this expected?

Reply:
I copied what was in https://github.com/SchillingRobotics/ros2_controllers-stogl/blob/eb1cd2b8693e947fc79335dac5adc9006b895b79/joint_trajectory_controller/src/trajectory.cpp#L632

Reply:
Additionally I can say that `update` in Ruckig lib, will most probably return `ruckig::Result::Working` since one needs multiple `update` calls  to get the `ruckig::Result::Finished` as is done in the while loops in the ruckig tests https://github.com/pantor/ruckig/blob/master/test/test-target.cpp#L141
The `ruckig joint limiter` is not using Ruckig to generate a trajectories, so I suppose a single `update` already applied the correct _profiles_ (not sure what multiple calls do to be honest)